### PR TITLE
Add 3 icons

### PR DESCRIPTION
**Pull request by Nido-nid**
## Description

I added icons of more Apps, I recently started to use and wondered, that they weren't implemented already.
I also thought, that there already was a calendar icon, but I was wrong. So I added one as well, but I noticed, that the drawable was updated. So it was just the SVG missing, I guess.
And lastly i corrected a typo from my last pull request in the grayscale_icon_map.xml. I forgot an n in the package name for the SBB Mobile app, so it said ch.sbb.mobile.adroid.b2c. So the icon wasn't implemented correctly.

https://www.dji.com/cn/downloads/djiapp/dji-ronin
https://play.google.com/store/apps/details?id=org.jitsi.meet
https://play.google.com/store/apps/details?id=de.ph1b.audiobook

## Type of change
:white_check_mark: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

## Fixes
* SBB Mobile (`ch.sbb.mobile.adroid.b2c` -> `ch.sbb.mobile.android.b2c`)

### Icons added
* DJI Ronin (`com.dji.ronin`)
* Jitsi Meet (`org.jitsi.meet`)
* Voice (`de.ph1b.audiobook`)

### Icons updated
* /e/ OS Calendar (`foundation.e.calendar`)

### CODE REVIEW

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -932,7 +935,7 @@
     <icon drawable="@drawable/samsung_internet_browser" package="com.sec.android.app.sbrowser" name="Samsung Internet Browser" />
     <icon drawable="@drawable/samsung_internet_browser_beta" package="com.sec.android.app.sbrowser.beta" name="Samsung Internet Browser Beta" />
     <icon drawable="@drawable/samsung_notes" package="com.samsung.android.app.notes" name="Samsung Notes" />
-    <icon drawable="@drawable/sbb_mobile" package="ch.sbb.mobile.adroid.b2c" name="SBB Mobile" />
+    <icon drawable="@drawable/sbb_mobile" package="ch.sbb.mobile.android.b2c" name="SBB Mobile" />
```
Comment (Goooler) [inline app/src/main/res/xml/grayscale_icon_map.xml:938]: Good catch! I've explicit the fix in 7405b8ddf34cdc99aee4971547c0fdc018698768.

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 2]
```diff
@@ -156,6 +156,7 @@
     <icon drawable="@drawable/calculator" package="com.oneplus.calculator" name="Calculator" />
     <icon drawable="@drawable/calculator" package="com.sec.android.app.popupcalculator" name="Calculator" />
     <icon drawable="@drawable/calculator_plus" package="com.gigaworks.tech.calculator" name="Calculator Plus" />
+    <icon drawable="@drawable/calendar" package="foundation.e.calendar" name="Calendar" />
```
Comment (Goooler) [inline app/src/main/res/xml/grayscale_icon_map.xml:159]: Calendar icons had been handled by #750.

Comment (Goooler) [inline app/src/main/res/xml/grayscale_icon_map.xml:159]: Calendar icons had been handled by #750, you should submit changes to Lawnchair like https://github.com/LawnchairLauncher/lawnchair/pull/3038.

**Review by Goooler — CHANGES_REQUESTED** [review]

**Review by Chefski — APPROVED** [review]
Hey! Icons look good. Next time try to keep the icon in strokes rather than converting to a shape. It just makes it easier to review.

As stated by @Goooler, Calendar icons are now in Lawnchair so I won't be reviewing it

File: app/src/main/res/xml/grayscale_icon_map.xml (hunk 2 above)
Comment (Goooler) [reply app/src/main/res/xml/grayscale_icon_map.xml]: Alright, I'm removing the calendar change now, you can send a new PR to Lawnchair.

**Comment by Nido-nid** [comment]
> Hey! Icons look good. Next time try to keep the icon in strokes rather than converting to a shape. It just makes it easier to review.
> 
> As stated by @Goooler, Calendar icons are now in Lawnchair so I won't be reviewing it

Problem is, the rounded ends of the strokes aren't taken over. In the SVG, they're there, but on the drawable preview they are squared. Is this a mistake, and they get implemented correctly anyway?

**Comment by Chefski** [comment]
> the rounded ends of the strokes aren't taken over. In the SVG, they're there, but on the drawable preview they are squared. Is this a mistake, and they get implemented correctly anyway?

You might have to show a screenshot whenever you see this again. It could be a problem when exporting as when I was reviewing these, the imported SVG was 1x1 in size so had to change to 192x192

**Comment by Nido-nid** [comment]
> > the rounded ends of the strokes aren't taken over. In the SVG, they're there, but on the drawable preview they are squared. Is this a mistake, and they get implemented correctly anyway?
> 
> You might have to show a screenshot whenever you see this again. It could be a problem when exporting as when I was reviewing these, the imported SVG was 1x1 in size so had to change to 192x192

So for instance, this was my initial jitsi icon drawn with paths. Designed in Affinity Designer. Exported in SVG with 192x192 pixels.
jitsi_path

But when converting it to a drawable, the rounded stroke-ends are squared. Could it be an issue with the program i use?
Bildschirmfoto vom 2022-12-08 11-24-59

**Comment by Chefski** [comment]
Not too sure why that happens... I suppose you can continue changing them to shapes or have two versions of the icon (one will remain in strokes and will be added to `svgs`, the other will be changed to a shape and used to create the drawable XML)

**Pull request merged**